index.html

```diff
@@ -1256,8 +1256,96 @@
       <section>
         <h3>Usage Patterns</h3>
 
-        <p class="issue">
-When verifiable claims are used can be used for correlation purposes.
+        <p>
+Despite the best efforts to assure privacy, the actual use of verifiable
+claims can potentially lead to de-anonymization and a loss of privacy.
+This correlation can occur:
+        </p>
+
+        <ol>
+          <li>
+When the same claim is presented to the same inspector more than once –
+that inspector could infer that the holder is the same individual.
+          </li>
+          <li>
+When the same claim is presented to different inspectors, and either those
+inspectors collude or a third party has access to transaction records from
+both inspectors – the observant party could infer that the individual
+presenting the claims is the same person at both services, i.e., the
+accounts are controlled by the same person.
+          </li>
+          <li>
+When the same subject ID of a claim refers to the same subject across
+presentations or inspectors. Even when different claims are presented,
+if the subject ID is the same, inspectors (and those with access to
+inspector logs) could infer that the holder of the claims is the same
+person.
+          </li>
+          <li>
+When the underlying information in a claim can be used to identify an
+individual across services – using information from other sources
+(including information provided directly by the user), inspectors can use
+the information inside the claim to correlate the individual with an
+existing profile. For example, if a holder presents claims that include
+zip code, age, and income, the inspector can potentially correlate the
+subject of that claim with an established profile which already contains
+the subject’s zip code, age, and income.
```

Comment:
We may want to use a more readily accessible method of correlation in the example: IP address and email address.